pimpmycause/profiles/models.py

```diff
@@ -1,26 +1,69 @@
 from __future__ import unicode_literals
 
 from django.db import models
-from django.contrib.auth.models import User
```

Comment:
No worries, if you check it out and run it. Go to `http://127.0.0.1:8000/admin`
This is where you can login with your user.

To create a user:
Do this https://docs.djangoproject.com/en/1.9/intro/tutorial02/#introducing-the-django-admin

Then login and have a look around admin, add new Pimp user or anything you like. Since the database is only local to you you can add/delete what you want in there.

Other things I used in this PR:

1. Django Registration Redux
- Github: https://github.com/macropin/django-registration,
- Docs: http://django-registration-redux.readthedocs.io/en/latest/
2. Django Forms: https://docs.djangoproject.com/en/1.9/topics/forms/
3. Django Custom user app: https://github.com/jcugat/django-custom-user